tests/testthat/test-unique-keys.R

```diff
@@ -0,0 +1,81 @@
+test_that("unique keys", {
+  skip_if_remote_src()
```

Comment:
Can we keep the snapshot and ensure stability by never printing the whole dm (or calling `dm_ptype()` before printing)?

Reply:
see 753a137931ccc8540f90bb29236e5dc0dbcc72aa